public/styles.css

```diff
@@ -0,0 +1,139 @@
+body {
```

Comment:
Great use of time being able to get this far in styling. Your app looks amazing!